app/views/home.njk

```diff
@@ -2,8 +2,52 @@
 
 {% block content %}
   <div class="govuk-grid-row">
-    <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-l">{{serviceName}}</h1>
+    <div class="govuk-grid-column-full">
+      <h1 class="govuk-heading-xl">{{serviceName}}</h1>
+      <div class="govuk-cards govuk-cards--2">
+        <div class="govuk-cards--2__item govuk-card--dashboard">
+          <h2 class="govuk-heading-m govuk-!-margin-bottom-2">Unattached reporting datasets</h2>
+          <p class="govuk-heading-xl govuk-!-margin-bottom-5">
+            {{ captureCount }}
+          </p>      
+          <ul class="govuk-list">
+            <li>
+              <a href="/capture" class="govuk-link">View unattached datasets</a>
+            </li>
+          </ul>
+        </div>
+        <div class="govuk-cards--2__item govuk-card--dashboard">
+          <h2 class="govuk-heading-m govuk-!-margin-bottom-2">Requests awaiting reporting data</h2>
+          <p class="govuk-heading-xl govuk-!-margin-bottom-5">
+            {{ enrichCount }}
+          </p>      
+          <ul class="govuk-list">
+            <li>
+              <a href="/enrich" class="govuk-link">View awaiting reporting data</a>
+            </li>
+          </ul>
+        </div>
+        <div class="govuk-cards--2__item govuk-card--dashboard">
+          <h2 class="govuk-heading-m govuk-!-margin-bottom-2">Requests awaiting quality check</h2>
+          <p class="govuk-heading-xl govuk-!-margin-bottom-5">
+            {{ qualityCheck }}
```

Comment:
Rename qualityCheckCount for consistency

Reply:
qualityCheck has been renamed to qualityCheckCount